tigerpath/static/app/js/tutorial.js

```diff
@@ -11,7 +11,7 @@ function startIntro() {
         intro: `You can search for courses you want to add to your schedule here. This can be by the course code or title.
         Try searching something!
```

Comment:
It seems that with the newer rewording, I've observed that users no longer take it to mean literally do it now.
This is not really related to this PR, bit it's a small enough change that it probably doesn't merit it's own.
```suggestion
         Try searching something now!
```